pandas/core/indexes/datetimes.py

```diff
@@ -175,6 +175,8 @@ def _new_DatetimeIndex(cls, d):
 class DatetimeIndex(DatelikeOps, TimelikeOps, DatetimeIndexOpsMixin,
                     Int64Index):
     """
+    Store timestamps using NumPy’s datetime64 dtype.
```

Comment:
You can ignore the comment about starting with a verb in this case, as this is the docstring of a class, and not a method or function.

I think someting lke `Datetime index using NumPy's...` would be more accurate.